tests/test-cases.js

```diff
@@ -1501,6 +1501,39 @@ describe('Parse Tests', function() {
 	}
 });
 
+var PARSE_ASYNC_ERROR_TESTS = [
```

Comment:
Why couldn't this test case go into the `PARSE_ASYNC_TESTS` test cases?

Reply:
No.

Reply:
Yes, please add this to the PARSE_ASYNC_TESTS

Reply:
Fixed